PyPoE/cli/core.py

```diff
@@ -119,6 +119,7 @@ def show_warning(self, *args, **kwargs):
 # Functions
 # =============================================================================
 
+quiet = False
```

Comment:
This is something I do not like the smell of. This is either a configuration parameter or a module constant, manipulating it outside of it's scope via `global` is not really what I would say is a good approach. Since in line 146 you are essentially redefining it and then again in 191.

A better approach is to either..

A) Just say "these bits are only needed to surface at a given situation and should be explicitly enabled for the run"
B) They are always enabled

And then control it via CLI argument which can then be passed around as a static value for the run.

Reply:
I think as with Zao's PR, we can merge this in, but really should focus on a tidy after the code's been put through a successful set of end to end runs

Reply:
Yeah that part's not great. l just did the minimum amount of effort to reduce spam so that l could see relevant errors.

l suspect it would be better to use python's logging module rather than a mix of print(), warnings.warn() and a custom console logger, but that would be a big change and might be better to do later in the patch cycle.

l've merged dev branch and there don't seem to be any problems; just running an export now to confirm.

Reply:
Yeah it's also not a priority, more of a refactor should you come back to this area next time. Thanks for all the good work